app/controllers/otp_setup_controller.rb

```diff
@@ -24,7 +24,7 @@ def create
   private
 
   def redirect_if_set_up
-    redirect_to dashboard_path if current_user.otp_enabled?
+    redirect_to organizations_path if current_user.otp_enabled?
```

Comment:
How would onboarding of users look like? Is it OK to validate that a user has at least one organization or is an admin? I think it would be odd to get redirected to the organizations listing page.

Reply:
On the organizations page is a redirect to the organization_dashboard page if the user only has/belongs to one organization. So technically there will be 2 redirects following each other. So UX wise nothing changes to a user being onboarded to one organization (see system spec).

Reply:
Although, the redirect in the organizations path does have the benefit of preventing a user who belongs to only 1 organization of visiting `/organizations` and setting an organization they do not belong to.

Reply:
I think there is no change needed, as currently user were always belonging only to one organization - and so there will always the redirect. As soon as we add the n:n relation here the code in the OrganizationsController has to be changed anyway.